docroot/themes/contrib/civictheme/civictheme_library/components/00-base/_variables.base.scss

```diff
@@ -271,8 +271,8 @@ $civictheme-typography-default: (
   'heading-6': ($civictheme-font-base-size, $civictheme-font-base-line-height * 1.625, 700, 'primary', 0),
   // Text.
   'text-extra-large': (
-    'xxs': ($civictheme-font-base-size * 1.25, $civictheme-font-base-line-height * 1.5, 400, 'primary', -0.1px),
-    'm': ($civictheme-font-base-size * 1.5, $civictheme-font-base-line-height * 1.5, 400, 'primary', 0)
+    'xxs': ($civictheme-font-base-size * 1.25, $civictheme-font-base-line-height * 2.125, 400, 'primary', -0.1px),
```

Comment:
please revert

Reply:
there is a `'quote'` typography style below

Reply:
@AlexSkrypnyk: Please consider this change in line-height as the content is not looking good with current line-height.